src/components/app-button/app-button.vue

```diff
@@ -35,6 +35,11 @@
 
 <script>
   export default {
+    setup() {
+      const { getDatoNuxtRoute } = useDatoNuxtRoute()
+
+      return { getDatoNuxtRoute }
+    },
```

Comment:
@Frankwarnaar this got me wondering why doesn't the `useDatoNuxtRoute` composable return the function `getDatoNuxtRoute` directly so that this setup code isn't needed?
Just did a quick test and it seems to work. :thought_balloon:

Reply:
Ah that's been a bit react-ish of me I think. I'm happy to using switch to returning the function directly, so please go ahead :)